terminal/charset.go

```diff
@@ -54,6 +54,10 @@ func scs1Handler(pty chan rune, terminal *Terminal) error {
 func scsHandler(pty chan rune, terminal *Terminal, which int) error {
 	b := <-pty
 
+	// Handler should lock the terminal if there will be write operations to any data read by the renderer
+	// terminal.Lock()
+	// defer terminal.Unlock()
```

Comment:
also here

Reply:
This is a reminder. It's left here intentionally.